LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
This file is already merged and shouldn't be part of your PR -> please rebase

Reply:
Hi honestly I am a noob with git, I try to rebase but I'm not able to fix this... Any help appreciated.

Reply:
yes ! I did it :)